src/librustc_codegen_llvm/Cargo.toml

```diff
@@ -13,7 +13,7 @@ test = false
 [dependencies]
 cc = "1.0.1" # Used to locate MSVC
 num_cpus = "1.0"
-rustc-demangle = "0.1.4"
+rustc-demangle = "0.1.14"
```

Comment:
This should be `0.1.15` too, right?

Reply:
Ah, oops, forgot to do that in one of the dozen rebases last night :D